src/scripts/main.js

```diff
@@ -1,3 +1,52 @@
 'use strict';
 
-// write code here
+const buttons = document.querySelectorAll('.button');
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+const table = document.querySelector('.field tbody');
+let totalRows = table.querySelectorAll('tr').length;
+let totalColumns = table.querySelectorAll('tr')[0].children.length;
+const max = 10;
+const min = 2;
+
+[...buttons].forEach(button => {
+  button.addEventListener('click', e => {
+    switch (e.target) {
+      case appendRow:
+        if (totalRows < max) {
```

Comment:
according to the task requirements buttons should be disabled in such cases:
![image](https://github.com/mate-academy/js_growth_table_DOM/assets/86923089/dc0fc2c0-f3d0-4a05-8970-b30ae3343aa4)